### PR TITLE
appcenter-dsym-upload 0.1.1

### PR DESCRIPTION
**Pull request by alexshikov**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/appcenter-dsym-upload/0.1.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+summary: |
+  Upload dSYM to Microsoft AppCenter
+description: "This step uploads dSYM to AppCenter using official CLI.\nThe step is
+  relevant only for iOS projects (including Xamarin.iOS).\n\nBy default $BITRISE_DSYM_PATH
+  env used to lookup for dSYM file.\nIf an archive step used as part of the workflow
+  then a default value can be used \nfor `dsym_path` variable\n\nIt is important to
+  [obtain API Token from AppCenter](https://appcenter.ms/settings/apitokens)\nto use
+  this step.\n"
+website: https://github.com/SeeD-Seifer/bitrise-step-appcenter-dsym-upload
+source_code_url: https://github.com/SeeD-Seifer/bitrise-step-appcenter-dsym-upload
```
Comment (viktorbenei) [inline steps/appcenter-dsym-upload/0.1.1/step.yml]: please fix this URL

File: steps/appcenter-dsym-upload/0.1.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+support_url: https://github.com/SeeD-Seifer/bitrise-step-appcenter-dsym-upload/issues
```
Comment (viktorbenei) [inline steps/appcenter-dsym-upload/0.1.1/step.yml]: please fix this URL

File: steps/appcenter-dsym-upload/0.1.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+title: AppCenter dSYM upload
+summary: |
+  Upload dSYM to Microsoft AppCenter
+description: "This step uploads dSYM to AppCenter using official CLI.\nThe step is
+  relevant only for iOS projects (including Xamarin.iOS).\n\nBy default $BITRISE_DSYM_PATH
+  env used to lookup for dSYM file.\nIf an archive step used as part of the workflow
+  then a default value can be used \nfor `dsym_path` variable\n\nIt is important to
+  [obtain API Token from AppCenter](https://appcenter.ms/settings/apitokens)\nto use
+  this step.\n"
+website: https://github.com/SeeD-Seifer/bitrise-step-appcenter-dsym-upload
```
Comment (viktorbenei) [inline steps/appcenter-dsym-upload/0.1.1/step.yml]: please fix this URL

**Comment by alexshikov** [comment]
Done. 
https://github.com/bitrise-io/bitrise-steplib/pull/1318

**Pull request closed without merging**